Gemfile

```diff
@@ -6,13 +6,14 @@ group :development, :test do
   gem 'puppetlabs_spec_helper',  :require => false
   gem 'puppet-lint',             :require => false
   gem 'simplecov',               :require => false
-  gem 'nokogiri',                '<= 1.5.10'
+  gem 'nokogiri',                :require => false
```

Comment:
This is no longer working for ruby 1.8.7.
Is newer nokogiri required for beaker?
In this case we should set a wrapper here for ruby version and install 1.5 if we are on ruby 1.8.7.